File: tests/conformance/fixtures/rust/27-agent-mandate.rs
use sykli::{Actor, ActorKind, EvidenceRequirement, Mandate, Pipeline, SuccessCriterion, TaskType};

fn main() {
    let mut p = Pipeline::new();
    p.task("implement")
        .run("go test ./...")
        .task_type(TaskType::Test)
        .success_criteria(&[SuccessCriterion::ExitCode(0)])
        .evidence_required(&[EvidenceRequirement::file_non_empty(
            "coverage",
            "coverage.out",
        )])
        .actor(Actor::new(ActorKind::Agent).id("claude"))
        .mandate(Mandate::new(&["sdk/**", "tests/conformance/**"]));
    p.emit();
}

File: tests/conformance/fixtures/typescript/27-agent-mandate.ts
import { Pipeline, fileEvidenceNonEmpty } from '../../../../sdk/typescript/src/index';

const p = new Pipeline();
p.task('implement')
  .run('go test ./...')
  .taskType('test')
  .successCriteria([{ type: 'exit_code', equals: 0 }])
  .evidenceRequired([fileEvidenceNonEmpty('coverage', 'coverage.out')])
  .actor({ kind: 'agent', id: 'claude' })
  .mandate({ scope: ['sdk/**', 'tests/conformance/**'] });
p.emit();
